microsoft/PowerToys
# [Launcher] Add missing Microsoft.Extensions.. dlls

## Summary of the Pull Request
* Add missing Microsoft.Extensions... dlls to the Launcher Wix installer Component.
* This should fix the Launcher .NET Core runtime exceptions.


## PR Checklist
* [] Tests added/passed
* [] Requires documentation to be updated
* [] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Description
This are the required DLLs, which are not part of .NET Core, but rather part of .NET Platform extensions:
* Microsoft.Extensions.Configuration.Abstractions.dll;
* Microsoft.Extensions.Configuration.Binder.dll;
* Microsoft.Extensions.Configuration.dll;
* Microsoft.Extensions.DependencyInjection.Abstractions.dll;
* Microsoft.Extensions.DependencyInjection.dll;
* Microsoft.Extensions.Logging.Abstractions.dll;
* Microsoft.Extensions.Logging.dll;
* Microsoft.Extensions.Options.dll;
* Microsoft.Extensions.Primitives.dll

They are used by NLog.Extensions.Logging, which is a dependency of Wox.Infraestructure that we could probably eliminate in the future. Their use is probably related to the Windows Compatibility Pack https://devblogs.microsoft.com/dotnet/announcing-the-windows-compatibility-pack-for-net-core/

## Validation Steps Performed
* Build an MSI
* Install Power Toys.
* Install the Desktop Runtime 3.1
* Start Power Toys
* Start launcher by using the shortcut.
* Launcher should appear on screen.
